infra/src/test/java/com/example/AppStackTest.java

```diff
@@ -17,7 +17,7 @@ void testAppStack() {
                 new File(AppStack.functionPath("function-java", Runtime.JAVA)).exists()) {
             AppStack stack = new AppStack(new App(), "TestMnTodoStack");
             Template template = Template.fromStack(stack);
-            template.hasResourceProperties("AWS::Lambda::Function", Collections.singletonMap("Handler", "io.micronaut.function.aws.proxy.MicronautLambdaHandler"));
+            template.hasResourceProperties("AWS::Lambda::Function", Collections.singletonMap("Handler", "io.micronaut.function.aws.proxy.payload1.ApiGatewayProxyRequestEventFunction"));
```

Comment:
@sdelamo Are we missing an assertion here, as this wasn't failing the build?

Reply:
./gradlew infra:test was failing without this change

Reply:
Weird... was that not run by the action?